addons/l10n_in/data/account.account.template.csv

```diff
@@ -5,8 +5,10 @@
 "p10051","SGST Receivable","10051","asset_current","l10n_in.indian_chart_template_standard","l10n_in.sgst_tag_account","False"
 "p10052","CGST Receivable","10052","asset_current","l10n_in.indian_chart_template_standard","l10n_in.cgst_tag_account","False"
 "p10053","IGST Receivable","10053","asset_current","l10n_in.indian_chart_template_standard","l10n_in.igst_tag_account","False"
-"p10057","Reverse Charge Tax Receivable","10057","asset_current","l10n_in.indian_chart_template_standard","","False"
-"p10054","TDS Receivable","10058","asset_current","l10n_in.indian_chart_template_standard","","False"
+"p10054","TDS Receivable","10054","asset_current","l10n_in.indian_chart_template_standard","","False"
+"p10057","RCM SGST Receivable","10057","asset_current","l10n_in.indian_chart_template_standard","","False"
```

Comment:
The spreadsheet says `100570 RCM IGST Receivable`, not SGST. Or is the idea to add RCM accounts and they don't have to necessarily match the requested coa changes?